src/components/tx/TxData.js

```diff
@@ -27,7 +27,10 @@ import { connect } from 'react-redux';
 
 
 const mapStateToProps = (state) => {
-  return { serverInfo: state.serverInfo };
+  return {
```

Comment:
Can't suggest on deleted lines, but can you refactor this piece to:

```suggestion
const mapStateToProps = (state) => ({
    nativeToken: state.serverInfo?.native_token ?? hathorLib.constants.DEFAULT_NATIVE_TOKEN_CONFIG,
    decimalPlaces: state.serverInfo?.decimal_places ?? hathorLib.constants.DECIMAL_PLACES,
});
```

Reply:
Done